test/CMakeLists.txt

```diff
@@ -47,7 +64,7 @@ if (WITH_TEST_GENERATORS)
   # and make an executable target for it named 'generator_foo'
   file(GLOB TESTS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/generator" "${CMAKE_CURRENT_SOURCE_DIR}/generator/*_jittest.cpp")
   foreach(test ${TESTS})
-    string(REPLACE "_jittest.cpp" "" name "generator_jit_${test}")
+    string(REPLACE "_jittest.cpp" "" name "generator_exec_jit_test_${test}")
```

Comment:
This is a dumb response, but...

Surely Windows systems in 2015 can be configured to avoid this Neanderthal limitation?